t/ui/12-needle-edit.t

```diff
@@ -171,18 +166,22 @@ sub change_needle_value($$) {
     is($decode_new_textarea->{area}[0]->{type}, "ocr", "type is ocr");
     $driver->double_click;    # the match type change back to match
 
+    unlike($driver->find_element('#change-match', 'css')->get_attribute('class'), qr/disabled/, "match level now enabled");
+
     # test match level
     $driver->find_element('#change-match', 'css')->click();
+    # wait for the fade
+    sleep 1;
```

Comment:
please don't. this simple sleep call is just wasting time. There should be a better way. Tests are taking way too long already.